src/pages/Profile/Profile.tsx

```diff
@@ -4,9 +4,23 @@ import { Link } from "react-router-dom";
 import { UserProfile } from "../../mockData/mockTribe";
 import Header from "../../components/Header/Header";
 import Navigation from "../../components/Navigation/Navigation";
+import { signOut } from "firebase/auth";
+import { auth } from "../../firebase";
+import { useNavigate } from "react-router-dom";
 
 const Profile = ({ user }: { user: UserProfile }) => {
   const { totalScore, img, name, bio, email } = user;
+  const navigate = useNavigate();
+
+  const handleSignOut = () => {
+    signOut(auth)
+      .then(() => {
+        navigate("/auth");
+      })
+      .catch(() => {
+        navigate("/ErrorPage");
+      });
```

Comment:
In the App we store the userID, can you set that back to null?
They also store the uuid in local storage, can you remove it as well?